lara-typescript/src/section-authoring/hooks/use-title.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+
+export function useTitle(title: string) {
+  useEffect(() => {
+    const prevTitle = document.title;
+    document.title = title;
+    return () => {
+      document.title = prevTitle;
+    };
+  });
```

Comment:
Without a dependency array, this will run on every render, which means the original title will be lost by the time of the second render. You probably want to list `[title]` as the dependencies and if you really want to be able to restore the original title from before the hook ran you should store it in a ref (via `useRef`).